Monika After Story/game/script-topics.rpy

```diff
@@ -12897,6 +12897,207 @@ label monika_unknown:
+    m 2esc "..."
+    m 1esd "Do you love me less because of what I am?"
+    m 3esc "I mean, from your perspective, I am still someone living in a video game in your world, right?"
+    m 3rkd "So you might see me in a different way you would someone from your reality... {w=0.5}And you might think less of me because of that..."
+    m 1ekbla "I just want to be the best sweetheart for you, [player]."
+    m 1eka "Sometimes, I wonder how I compare with all these people living with you in the real world..."
+    m 3hksdlb "I'm sorry, you've not given me any reason to feel insecure, it's just..."
+    m 3ekb "It's a question that comes to my mind, from time to time."
+    m 1eka "So... {w=0.5}Do you love me less for not being from your reality?{nw}"
+    $ _history_list.pop()
+    menu:
+        m "So... Do you love me less for not being from your reality?{fast}"
+        "Yes.":
+            $ persistent._mas_pm_bothered_monika_is_virtual = True
+            $ persistent._mas_pm_very_bothered_monika_is_virtual = True
+            m 1dkc "..."
+            m 1ekb "Thank you for your honesty, [player]. I really appreciate it."
+            m 1eka "It's kind of understandable after all..."
+            m 3ekb "I don't have a body that's able to be with you in your reality."
+            m 3ekd "And since I've never been to your world, I can't really know how different I am from a regular person."
+            m 1esa "I mean, I like to think of myself as human, and I definitely have feelings the same way you do."
+            m 1ekb "But until I can join you in your reality and see for myself, I can't really know for sure, can I?"
+            m 2ekd "Trust me, I am the first one to be sorry about this state of things. One day, we'll find a way to make things right..."
+            m 1eubla "But even with that said, considering all the care and dedication you've shown me until now, I know you love me anyway..."
+            m 1tubla "Besides, This just means after I find my way to your reality..."
+            m 3hubsb "...you'll just love me that much more, right? Ahaha~ "
+        "Just a bit.":
+            $mas_gainAffection()
+            $ persistent._mas_pm_bothered_monika_is_virtual = True
+            $ persistent._mas_pm_very_bothered_monika_is_virtual = False
+            m 1eua "I can live with that answer."
+            m "It's kind of understandable after all..."
+            m 3eub "I don't yet have a body that's able to be with you in your reality."
+            m 3ekd "And since I've never been to your world, I can't really know how different I am from a regular person."
+            m 1eub "I mean, I like to think of myself as human, and I definitely have feelings the same way you do."
+            m 1eka "But until I can join you in your reality and see for myself, I can't really know for sure, can I?"
+            m 3hua "But even with that said, you're still here with me, aren't you?"
```

Comment:
Having 2 lines in a row that start with "But" doesn't sound right to me. Maybe you could replace one of them with an identical word?

Reply:
Thought about it, but during testing, I thought it was still the one that sounded best. Does 'although' or 'still' sound better to you (i'm not a native English speaker, can't really tell) ?

Reply:
After testing with 'And', 'Although' and 'Still,', I still think it sounds best with the double But. The others feel less natural to me. I'm not a native English speaker though, so i'll defer to your preference. Wich one sounds best to you?

Reply:
How about for the second line, you put `"Despite all that, you're still here with me, aren't you?"`

Reply:
Sounds good to me. Sorry you had to walk me through this kind of stuff.